packages/design-tokens/utils/theme-prop-type.ts

```diff
@@ -164,7 +164,7 @@ export const themePropType = PropTypes.shape({
       three: typePropType,
       four: typePropType,
     }).isRequired,
-    subHeadline: PropTypes.shape({ typePropType }).isRequired,
+    subHeadline: PropTypes.shape({ ...typePropType }).isRequired,
```

Comment:
```suggestion
    subHeadline: PropTypes.shape(typePropType).isRequired,
```

Reply:
Hm, apparently TypeScript is unhappy about this 🙄

Reply:
Feel free to revert.

Reply:
It should be just `subHeadline: typePropType` 😅